lib/index.js

```diff
@@ -29,7 +29,40 @@ var Klaviyo = module.exports = integration('Klaviyo')
  * @api public
  */
 
-Klaviyo.prototype.identify = request('/identify');
+Klaviyo.prototype.identify = function(msg, fn) {
+  var batch = new Batch();
+  var self = this;
+
+  batch.throws(true);
+
+  this
+    .get('/identify')
+    .query({ data: new Buffer(JSON.stringify(msg.peopleData)).toString('base64') })
+    .end(function(err, res){
+      if (err) return fn(err);
+
+      // 1 = success 0 = failure
+      if (res.text != '1') {
+        err = self.error('bad response');
+        return fn(err, res);
+      }
+      // // https://www.klaviyo.com/docs/api/lists
+      if (msg.listData) {
+        batch.push(function (done){
```

Comment:
and is the call to /list dependent on having successfully identified? if not, we could fire off both requests at once using a `Batch`. Otherwise, just executing the second request in the callback to the first is fine to do without batching